nav2_bt_navigator/behavior_trees/follow_point.xml

```diff
@@ -10,13 +10,13 @@
       <RateController hz="1.0">
         <Sequence>
           <GoalUpdater input_goal="{goal}" output_goal="{updated_goal}">
-            <ComputePathToPose goal="{updated_goal}" path="{path}" planner_id="{selected_planner}" error_code_id="{compute_path_error_code}"/>
```

Comment:
Blanketly: make sure you update all the BTs. I think you have, but please recheck if any are missing (since those will cause real issues if a user tries to use it without that set)

Reply:
Added tool/checkBehaviourTree.py to verify that every element that has an error_code_id attribute, also has an error_msg element.  And that the prefix in the values of each of these elements is identical.